tests/ulyanov_et_al_2016/test_loss.py

```diff
@@ -1,32 +1,155 @@
+import itertools
+
+import pytest
+
 import pytorch_testing_utils as ptu
 from torch.nn.functional import mse_loss
 
+from pystiche import gram_matrix, ops
 from pystiche.image import extract_batch_size
+from pystiche.loss import PerceptualLoss
 from pystiche_papers.ulyanov_et_al_2016 import loss
 
 
-def test_ulyanov_et_al_2016_content_loss(
-    subtests, multi_layer_encoder_with_layer, batch_image
+def test_UlyanovEtAl2016FeatureReconstructionOperator(
+    subtests, multi_layer_encoder_with_layer, target_image, input_image
 ):
     multi_layer_encoder, layer = multi_layer_encoder_with_layer
     encoder = multi_layer_encoder.extract_encoder(layer)
-    target_image = batch_image
-    input_image = batch_image.flip((0,))
     target_enc = encoder(target_image)
     input_enc = encoder(input_image)
-    batch_size = extract_batch_size(batch_image)
 
-    configs = ((True, batch_size), (False, 1.0))
-    for impl_params, extra_batch_size_mean in configs:
-        op = loss.ulyanov_et_al_2016_content_loss(
+    for impl_params in (True, False):
+        with subtests.test(impl_params=impl_params):
+            op = loss.UlyanovEtAl2016FeatureReconstructionOperator(
+                encoder, impl_params=impl_params,
+            )
+            op.set_target_image(target_image)
+            actual = op(input_image)
+
+            score = mse_loss(input_enc, target_enc)
+
+            desired = score / extract_batch_size(input_enc) if impl_params else score
```

Comment:
By default the `input_image` and `target_image` have `batch_size==1`. Thus if we don't increase it manually, this will work regardless of a "correct" implementation.